.github/workflows/ci.yml

```diff
@@ -321,20 +322,35 @@ jobs:
         run: |
           ./build_tools/scripts/check_cuda.sh
           ./build_tools/scripts/check_vulkan.sh
-      - name: "Downloading build dir archive"
-        run: gcloud storage cp "${BUILD_DIR_GCS_ARTIFACT}" "${BUILD_DIR_ARCHIVE}"
-      - name: "Extracting build dir archive"
-        run: tar -xf "${BUILD_DIR_ARCHIVE}"
-      - name: "Testing all"
-        env:
-          IREE_NVIDIA_SM80_TESTS_DISABLE: 1
-          IREE_MULTI_DEVICE_TESTS_DISABLE: 1
-          IREE_CTEST_LABEL_REGEX: ^requires-gpu|^driver=vulkan$|^driver=cuda$
+      - name: "Downloading install dir archive"
+        run: gcloud storage cp "${INSTALL_DIR_GCS_ARTIFACT}" "${INSTALL_DIR_ARCHIVE}"
+      - name: "Extracting install dir archive"
+        run: tar -xf "${INSTALL_DIR_ARCHIVE}"
+      - name: "Configuring runtime build"
         run: |
           ./build_tools/github_actions/docker_run.sh \
-              --env IREE_NVIDIA_SM80_TESTS_DISABLE \
-              --env IREE_MULTI_DEVICE_TESTS_DISABLE \
-              --env IREE_CTEST_LABEL_REGEX \
+              gcr.io/iree-oss/nvidia@sha256:892fefbdf90c93b407303adadfa87f22c0f1e84b7e819e69643c78fc5927c2ba \
+              cmake -B ${BUILD_DIR} -S . -G Ninja \
+                  -DCMAKE_BUILD_TYPE=RelWithDebInfo \
+                  -DIREE_BUILD_COMPILER=OFF \
+                  -DLLVM_EXTERNAL_LIT=$PWD/third_party/llvm-project/llvm/utils/lit/lit.py \
```

Comment:
`/usr/bin/python3.9: can't open file '/runner-root/actions-runner/_work/iree/iree/third_party/llvm-project/llvm/utils/lit/lit.py': [Errno 2] No such file or directory`

https://github.com/openxla/iree/actions/runs/7718944930/job/21041521228?pr=16216#step:10:941

(╯°□°)╯︵ ┻━┻ 

tempted to lift this out of `iree_llvm.cmake` / include it when importing tools somewhere: https://github.com/openxla/iree/blob/91b3ed5274f81a9fd414e3abd0c57a77074c420b/build_tools/cmake/iree_llvm.cmake#L75

Reply:
@stellaraccident do you know if we could include `lit.py` in our packages (iree-dist and/or CMake install)?

I'd like for test CI jobs to be able to use the lit runner without needing to configure the full compiler (or ideally without even checking out the LLVM submodule).

Context:

* https://github.com/openxla/iree/blob/dc4e59ec0c513872e320836142e79707edf28b1d/build_tools/cmake/iree_llvm.cmake#L101-L108
* Related discussion: https://github.com/openxla/iree/pull/12256#discussion_r1115001090

Reply:
Seems like I can install lit from https://pypi.org/project/lit/. That gets a little tricky when Docker is involved too, but I found a few setups that work.